crates/edr_provider/src/data.rs

```diff
@@ -1736,12 +1736,16 @@ where
     > {
         let cfg_env = self.create_evm_config_at_block_spec(block_spec)?;
 
+        let mut runtime_observer = RuntimeObserver::new(observability::Config {
+            call_override: None,
```

Comment:
Is `call_override: None` correct here? If yes, could you leave a comment in the code please explaining why?

Reply:
From a product perspective, we didn't support mocking for `debug_traceCall`. I'm not 100% sure why, but I assume because you're trying to trace a call the way it was originally executed.

Now that I'm adopting the `RuntimeObserver` instead of the `TraceCollector` - in order to support code coverage collection - I need to ensure we explicitly disable call override, even if it's generally turned on.

Reply:
@fvictorio is my understanding of why `debug_traceCall` doesn't support mocking correct?

Reply:
I don't remember tbh. I don't see any reason not to support it: I can imagine someone saying "ok, but how would have this code been executed/traced if \<some override>".

Reply:
I added an issue to track this feature: https://github.com/NomicFoundation/edr/issues/881